docs/content/guides/rows/row-sorting.md

```diff
@@ -453,163 +1184,2251 @@ import 'handsontable/dist/handsontable.full.min.css';
+:::
 
-::: tip
-Hooks are also run when you use the [`clearSort`](@/api/columnSorting.md#clearsort) method or provide a configuration that won't be processed, causing validation to fail.
 :::
 
-## Plugin API
+## Import the sorting module
 
-List of methods exposed by the plugin:
+If you're using Handsontable through [modules](@/guides/tools-and-building/modules.md),
+you need to do the following before you can [enable sorting](#enable-sorting):
 
-* [`clearSort`](@/api/columnSorting.md#clearsort)
-* [`getSortConfig`](@/api/columnSorting.md#getsortconfig)
-* [`isSorted`](@/api/columnSorting.md#issorted)
-* [`setSortConfig`](@/api/columnSorting.md#setsortconfig)
-* [`sort`](@/api/columnSorting.md#sort)
+```js
+// import the base module
+import Handsontable from 'handsontable/base';
 
-## Related articles
+// import Handsontable's CSS
+import 'handsontable/dist/handsontable.full.min.css';
+
+// import the sorting plugins
+import {
+  registerPlugin,
+  ColumnSorting,
+  MultiColumnSorting,
+} from 'handsontable/plugins';
 
-### Related guides
+// register the sorting plugins
+registerPlugin(ColumnSorting);
+registerPlugin(MultiColumnSorting);
+```
 
-- [Column sorting](@/guides/columns/column-sorting.md)
+## API reference
 
-### Related blog articles
+| Plugins                                                                                          | Options                                                                                                          | Handsontable's hooks                                                                                         |
+| ------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------ |
+| [`ColumnSorting`](@/api/columnSorting.md)<br>[`MultiColumnSorting`](@/api/multiColumnSorting.md) | [`columnSorting`](@/api/options.md#columnsorting)<br>[`multiColumnSorting`](@/api/options.md#multicolumnsorting) | [`afterColumnSort`](@/api/hooks.md#aftercolumnsort)<br>[`beforeColumnSort`](@/api/hooks.md#beforecolumnsort) |
 
-- [Feature spotlight: Multi-column sorting](https://handsontable.com/blog/feature-spotlight-multi-column-sorting)
+## Troubleshooting
 
-### Related API reference
+Didn't find what you need? Try this:
 
-- Configuration options:
-  - [`columnSorting`](@/api/options.md#columnsorting)
-  - [`multiColumnSorting`](@/api/options.md#multicolumnsorting)
-- Hooks:
-  - [`afterColumnSort`](@/api/hooks.md#aftercolumnsort)
-  - [`beforeColumnSort`](@/api/hooks.md#beforecolumnsort)
-- Plugins:
-  - [`ColumnSorting`](@/api/columnSorting.md)
-  - [`MultiColumnSorting`](@/api/multiColumnSorting.md)
+- View [GitHub issues](https://github.com/handsontable/handsontable/labels/Column%20sorting) related to sorting.
```

Comment:
1. It suggests that the sorting feature has "issues" (meaning bugs?). I would replace it with "View related topics on GitHub" or similar
2. I would remove the full-stop symbols from the end of sentences because they are, in fact, not sentences.
3. Link should be CTAs (call to action), so in a sentence "Report a new GitHub issue" the link should be applied to CTA words, e.g. [Report an issue]() on GitHub; [Ask a question]() on StackOverflow; [Start discussion]() on forum; [Contact our technical support]() to get help